doc/sphinxext/github_link.py

```diff
@@ -40,12 +40,13 @@ def _linkcode_resolve(domain, info, package, url_fmt, revision):
         return
 
     class_name = info['fullname'].split('.')[0]
-    if type(class_name) != str:
```

Comment:
I have removed these lines because they were intended only for Python 2, and, as far as I am concerned, scikit-learn does not support Python 2 anymore.